### DISCUSSION TITLE
Windows XP

### DISCUSSION THREAD

**Discussion by gelbehexe** [FaithBeam/Sims-1-Complete-Collection-Widescreen-Patcher#48] [Q&A]
Hello, can I use this tool with Windows XP - (the game runs in a virtual machine). What release should I choose?

Thanks in Advanced

**Comment by FaithBeam** [comment]
@gelbehexe I just released the legacy branch and created a release for it. Try this release: https://github.com/FaithBeam/Sims-1-Complete-Collection-Widescreen-Patcher/releases/tag/1.23.0-legacy.

I do not have a windows XP install so I have not tested in that environment. It targets .NET Framework 4.0 so it should be fine. Ignore the dgvoodoo2 checkbox because I don't believe that will work on Windows XP. When you click patch, the patcher will look like it has frozen (potentially minutes depending on the speed of your system), give it some time and it should eventually go through.

And make sure to install .NET Framework 4 because it is not installed in XP by default: https://www.microsoft.com/en-US/Download/confirmation.aspx?id=17718

**Reply by gelbehexe** [reply]
Thank you very much! I will try it tomorrow, for it is late in Germany :-)

**Comment by FaithBeam** [comment]
@gelbehexe 
I actually fixed the patcher for Windows XP. The previous release didn't work and I assumed it did. I've confirmed that this works: https://github.com/FaithBeam/Sims-1-Complete-Collection-Widescreen-Patcher/releases/tag/1.24.0-legacy
